data/remnant missions.txt

```diff
@@ -649,66 +644,135 @@ mission "Remnant: Catalytic Ramscoop"
+		event "Remnant: Plasma Cannon Available" 80 160
 		conversation
 			`As <ship> settles onto the pad closest to the research facility, you can see the weapons specialist checking over a large cannon on a platform beneath a nearby Albatross. When he sees you starting to unload the two plasma cannons, he snaps the casing closed and gestures to someone out of sight. Within moments both him and an empty flatbed are there to pick up the weapons.`
 			`	"Thank you for your help," he says. "Our Inhibitors provide us with something of a mobility advantage against the Korath, but they aren't particularly effective at disabling them. These plasma cannons will open new avenues of research for us." He pauses, then continues. "I also included a bonus for you. It is a 'finder's fee,' as we always appreciate new tech."`
 			`	The weapon technician finishes strapping down the plasma cannons and looks up to see another Remnant approaching. He gives the new Remnant a respectful dip of his head that almost seems like a bow before turning to you and chanting, "Ah, Captain <last>, have you met Tali? She is the engineering prefect here on Viminal. She asked me to let her know when or if you managed to procure the new tech for me."`
+			branch
+				has "Remnant: Catalytic Ramscoop: done"
+			`	Tali steps forward to face you. "Well met once again, <first>," she trills. "Your fresh actions set songs in motion that have long been in caesura. I have a few things to finish up right now, but I hope to work with you soon." She gives you a polite nod and heads back the way she came.`
+			branch
+				has "Remnant: Heavy Laser: done"
+			`	Tali steps forward to face you. "Well met once again, <first>," she trills. "Your fresh actions set songs in motion that have long been in caesura. I have a few things to finish up right now, but I hope to work with you soon." She gives you a polite nod and heads back the way she came.`
+			
 			`	Tali steps forward to face you. "Well met, newcomer among the Remnant," she trills. "Fresh blood amongst us sets songs in motion that have long been in caesura. I have a few things to finish up right now, but I hope to work with you soon." She gives you a polite nod and heads back the way she came.`
 
 
 
+mission "Remnant: Tali Log Entry"
+	name "Tali Log Entry"
+	source
+		government "Remnant"
+	invisible
+	to offer
+		or
+			has "Remnant: Plasma Weapons: done"
+			has "Remnant: Catalytic Ramscoop: done"
+			has "Remnant: Heavy Laser: done"
+	on offer
+		log "People" "Tali" `As an engineering prefect, Tali is the foremost authority among the Remnant on salvaged technology and a senior leader in their reverse engineering program.`
+		conversation
+			`A message pops up on your communicator from Tali. The message reads:`
+			`	We are always on the lookout for new technology, no matter the source. If you find any, just get in touch with me.`
+			`				Onwards,`
+			`						Prefect Tali`
+				decline
```

Comment:
There doesn't necessarily need to be a message to the player once they get the log entry. I'd find it odd to meet Tali, go to the spaceport, then receive a message from her. I'd 1) remove this message and have this info be something Tali tells you when you meet her and 2) remove the source from this mission so that the log entry is offering anywhere and 3) make the mission offer on landing so that you get the log entry immediately instead of upon entering a spaceport.